api/src/services/email.service.ts

```diff
@@ -560,6 +560,100 @@ export class EmailService {
     );
   }
 
+  public async lotteryReleased(
+    user: User,
+    listingInfo: listingInfo,
+    emails: string[],
+    appUrl: string,
+  ) {
+    try {
+      const jurisdiction = listingInfo.juris
+        ? await this.getJurisdiction([{ id: listingInfo.juris }])
+        : user.jurisdictions[0];
+      void (await this.loadTranslations(jurisdiction));
+      await this.send(
+        emails,
+        jurisdiction.emailFromAddress,
+        this.polyglot.t('lotteryReleased.header', {
+          listingName: listingInfo.name,
+        }),
+        this.template('lottery-released')({
+          appOptions: { listingName: listingInfo.name },
+          appUrl: appUrl,
+          listingUrl: `${appUrl}/listings/${listingInfo.id}`,
+        }),
+      );
+    } catch (err) {
+      console.log('lottery released email failed', err);
+      throw new HttpException('email failed', 500);
+    }
+  }
+
+  public async lotteryPublishedAdmin(
+    user: User,
+    listingInfo: listingInfo,
+    emails: string[],
+    appUrl: string,
+  ) {
+    try {
+      const jurisdiction = listingInfo.juris
+        ? await this.getJurisdiction([{ id: listingInfo.juris }])
+        : user.jurisdictions[0];
+      void (await this.loadTranslations(jurisdiction));
+      await this.send(
+        emails,
+        jurisdiction.emailFromAddress,
+        this.polyglot.t('lotteryPublished.header', {
+          listingName: listingInfo.name,
+        }),
+        this.template('lottery-published-admin')({
+          appOptions: { listingName: listingInfo.name, appUrl: appUrl },
+        }),
+      );
+    } catch (err) {
+      console.log('lottery published admin email failed', err);
+      throw new HttpException('email failed', 500);
+    }
+  }
+
+  /**
+   *
+   * @param emails a key in LanguagesEnum to a list of emails of applicants who submitted in that language
+   */
+  public async lotteryPublishedApplicant(
+    listingInfo: listingInfo,
+    emails: { [key: string]: string[] },
+  ) {
+    try {
+      const jurisdiction = await this.getJurisdiction([
+        { id: listingInfo.juris },
+      ]);
+
+      for (const language in emails) {
+        void (await this.loadTranslations(null, language as LanguagesEnum));
+        await this.send(
+          emails[language],
+          jurisdiction.emailFromAddress,
+          this.polyglot.t('lotteryAvailable.header', {
+            listingName: listingInfo.name,
+          }),
+          this.template('lottery-published-applicant')({
+            appOptions: {
+              listingName: listingInfo.name,
+              appUrl: jurisdiction.publicUrl,
+            },
+            appUrl: jurisdiction.publicUrl,
+            notificationsUrl: 'https://www.exygy.com',
+            helpCenterUrl: 'https://www.exygy.com',
```

Comment:
Is the exygy url just a placeholder url? Or is it intended to be here because in core we don't have a specific url?

Reply:
Sort of both, we don't have anything to set this to in core

Reply:
I wonder if we should make those parts of the email conditional on if the urls exist. So for example if `notificationsUrl` doesn't exist we won't show the sentence "You can sign up to receive notifications of new application opportunities here"

Reply:
I think my only light concern with that is that I think ideally the feature is fully demo-able and testable in core, and then core wouldn't reflect the full emails.

Reply:
In doorway will this be determined by the env variable? I'm assuming the notifications url will match the `NOTIFICATIONS_SIGN_UP_URL` variable in public?

I'm ok with this being here for now. Maybe we should put a comment here that these are just placeholder urls. I'd prefer to have these urls not hardcoded here long term when it will have to be dynamic for each jurisdiction/environment. But to get this in a demo-able state no need to update that immediately.

Reply:
Great q on the public site env variable I will ask! So your ideal long term is that these pieces won't be set in core and won't show? Will add a comment here